### PR TITLE
1.0.0 doc changes

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Release Information
+
+- **Version**: 1.0.0
+- **Certified**: No
+- **Publisher**: Fortinet
```
Comment (cs-suyog-jinturkar) [inline README.md:5]: Certified yes

File: docs/usage.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+    "version": "4.0.0",
+    "configName": [ "demo-1" ]
+  }
+]
+```
+
+| Field Type | Key Value Pair                         | Description                                                                                                                                                           |
+|:-----------|:---------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **string** | `"name": "ConnectorName"`              | This is a *mandatory* field. If left blank, the connector appears under **Invalid** when specified under *Include* list and ignored when specified in *Exclude* list. |
+| **string** | `"version": "VersionNumber"`           | This is an *optional* field. If left blank, it fetches status of all available versions                                                                               |
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md]: This denotes Connector version. This is an optional field.........

Same can be done for other fields as well.

Comment (cs-suyog-jinturkar) [reply docs/usage.md]: Can we give Version example as well here like 2.0.0 and not v2.0.0 as it's mentioned in docs like this.

File: docs/usage.md [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+| **array**  | `"configName": ["config1", "config2"]` | This is an *optional* field. If left blank, it fetches status of all available configurations.                                                                        |
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md]: Available connector configuration

File: docs/usage.md [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+[
+  {
+    "name": "VirusTotal",
+    "version": "3.0.0",
+    "configName": [ "vt-1, "vt-2" ]
+  }
+]
+```
+
+In such a case, the notification email displays the health status of the specified connector(VirusTotal), its specified version(3.0.0), and its mentioned configuration(vt-1 and vt-2).
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md]: How does the health status notification will start? Does user will need to configure a schedule for it? If yes,How and it's frequency

**Review by cs-suyog-jinturkar — COMMENTED** [review]
Added few review comments

**Comment by cs-abhishek-shukla** [comment]
@cs-suyog-jinturkar, the feedback has been incorporated. Thanks.

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**